plugins/email_service/app/helpers/email_service/application_helper.rb

```diff
@@ -1164,28 +1460,30 @@ def nebula_status
           return JSON.parse(nebula_details).has_key?("error") ? "ERROR" : "" rescue nil
         end
       end
+      
       return status ? status : nil
-      # {"production"=>true, "status"=>"GRANTED", "security_attributes"=>"security officer David Halimi (I349172), environment DEV, valid until 2022-11-22", "compliant"=>true}
-      # debugger
+
     end
 
     # check if cronus service is enabled for the project
-    def nebula_active? 
+    def nebula_active?
+      
       @nebula_active ||= nebula_details && nebula_details["status"] == "GRANTED" ? true : false
-      # false
+
     end
 
     def get_nebula_uri(provider = nil, custom_url = nil)
-      unless custom_url == nil 
+
+      unless custom_url == nil
         @nebula_url ||= URI("#{custom_url}/v1/#{provider}/#{project_id}")
       else
-        # TODO : findout if URI is valid
         provider = "aws" unless provider
         @nebula_url ||= URI("https://nebula.#{cronus_region}.cloud.sap/v1/#{provider}/#{project_id}")
```

Comment:
please change all endpoint references to use the one from catalog and not hardcoded.

Reply:
done